doc/source/installation.rst

```diff
@@ -63,23 +64,26 @@ For Ubuntu, run the following commands:
 .. code-block:: bash
 
   sudo apt-get update
-  sudo apt-get install -y cmake pkg-config build-essential autoconf curl libtool unzip flex bison psmisc python # we install python here because python2 is required to build the webui
+  sudo apt-get install -y build-essential curl unzip psmisc python # we install python here because python2 is required to build the webui
+
+  # Install Bazel.
+  ./ci/travis/install-bazel.sh
```

Comment:
This only works once ray is cloned and once we cd'ed into the ray directory.

Reply:
Whoops, should be fixed now. Thanks!